src/Halogen/Menu/Component.purs

```diff
@@ -0,0 +1,124 @@
+module Halogen.Menu.Component where
```

Comment:
That's the one. It'll rexport whatever you import from those modules, so a corresponding import is required, and therefore probably shouldn't be a selective import either. You could import a bunch of stuff qualified `as Rexports` and then `export Rexports` as an alternative also.